src/mixins/Projector.ts

```diff
@@ -109,16 +109,16 @@ export interface ProjectorMixin<P> {
 }
 
 export function ProjectorMixin<P, T extends Constructor<WidgetBase<P>>>(Base: T): T & Constructor<ProjectorMixin<P>> {
-	class Projector extends Base {
+	abstract class Projector extends Base {
 		public projectorState: ProjectorAttachState;
-		public properties: Readonly<P> & Readonly<ProjectorProperties>;
 
-		private _root: Element;
+		private _root: Element = document.body;
```

Comment:
At some point we should move things like this to use a reference to global, but it's consistent with what we're doing elsewhere.